arrow/src/main/java/org/apache/iceberg/arrow/vectorized/ArrowReader.java

```diff
@@ -91,9 +92,8 @@
  *   <li>Columns with constant values are physically encoded as a dictionary. The Arrow vector type
  *       is int32 instead of the type as per the schema. See
  *       https://github.com/apache/iceberg/issues/2484.
- *   <li>Data types: {@link Types.ListType}, {@link Types.MapType}, {@link Types.StructType}, {@link
- *       Types.FixedType} and {@link Types.DecimalType} See
- *       https://github.com/apache/iceberg/issues/2485 and
+ *   <li>Data types: {@link Types.ListType}, {@link Types.MapType}, {@link Types.StructType}, and
+ *       {@link Types.DecimalType} See https://github.com/apache/iceberg/issues/2485 and
```

Comment:
decimal type support also exists already, so we can remove that comment

Reply:
Yes, that's right, I remove this comment in my other PR: #13562